action-ssh-to-runner/disconnect

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -x -e
+
+kill -SIGKILL $(cat /tmp/tmate.pid)
```

Comment:
> We should also prevent the [action-delay-job-completion](https://github.com/scality/actions/blob/main/action-delay-job-completion/action.yaml) from running

Good point. We should also prevent it in case the "Connect to SSH" step timeouts because noone connected.

My thought was to check for the existence of `/tmp/tmate.pid`, but I intent to do this in a separate ticket/PR.

Reply:
Checking if `tmate` is running is indeed better.
Will you use the same ticket for the other PR ? If not please create a new one.

Reply:
I created https://scality.atlassian.net/browse/OS-668